client/src/components/TracerProfileForm.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <form novalidate>
+    <md-field>
+      <label>Tracer Profile</label>
+      <md-select v-model="model.profile_model">
+        <md-option
+          v-for="(value, choice) in choices"
+          :key="choice"
+          :value="value">
+          {{choice}}
+        </md-option>
+      </md-select>
+    </md-field>
+    <div class="md-layout-item">
+      <div v-for="(value, param) in model.profile_params"
+        :key="param"
+      >
+      <double-field
+        v-if="typeof value === 'number'"
+        :value="value"
+        :param="param"
+        range=false
+        :placeholder="String(defaults[param])"
+        v-model="model.profile_params[param]"/>
+      <md-checkbox
+        v-if="typeof value === 'boolean'"
+        class="md-primary"
+        v-model="model.profile_params[param]"
+      >
```

Comment:
This has a little bug ( try typing f into the text field ) mainly because doublefield saves values as a string for validation purposes. 

Simple fix would be to do v-if="param === 'interop'" for the checkbox and v-else otherwise.